botocore/httpsession.py

```diff
@@ -37,8 +38,12 @@
     from ssl import OP_NO_TICKET, PROTOCOL_TLS_CLIENT
 
 try:
-    # Always import the original SSLContext, even if it has been patched
-    from urllib3.contrib.pyopenssl import orig_util_SSLContext as SSLContext
+    with warnings.catch_warnings():
```

Comment:
```suggestion
    # pyopenssl will be removed in urllib3 2.0, we'll fall back to ssl_ at that point.
    # This can be removed once our urllib3 floor is raised to >=2.0.
    with warnings.catch_warnings():
```